common/src/main/java/org/apache/drill/exec/metrics/CpuGaugeSet.java

```diff
@@ -49,6 +51,15 @@ public CpuGaugeSet() {
     metric.put("drillbit.uptime", new DrillbitUptime(rtMXBean));
     return metric;
   }
+
+  private static OperatingSystemMXBean getOSMXBeanForCpuMetrics() {
+    try {
+      return (OperatingSystemMXBean) ManagementFactory.getOperatingSystemMXBean();
+    } catch (Exception ex) {
```

Comment:
Not sure how it can throw other exception instead of class cast. I liked version with more narrow exception better.